### DISCUSSION TITLE
How to change color of selected option in MultiSelect with props disableSelectedItemFiltering

### DISCUSSION THREAD

**Discussion by ks-himanshub** [mantinedev/mantine#4179] [Q&A]
I am not able to find any style props to change color of selected components, I changed color forcefully by to logic of selected options, but that is overriding the feature of highlight option on keyboard navigation or mouse hover

**Comment by rtivital** [comment]
Use `data-active` attribute, it is similar to Select component – https://mantine.dev/core/select/#styles-api
